stock_picking_consolidation_priority/tests/test_consolidation_priority.py

```diff
@@ -298,6 +298,16 @@ def assert_priority(self, changed_moves, unchanged_moves):
                 ),
             ),
         )
+        changed_pickings = changed_moves.picking_id
+        unchanged_pickings = unchanged_moves.picking_id
+        changed_ok = all(
+            picking.priority == expected_priority for picking in changed_pickings
+        )
+        unchanged_ok = all(
+            picking.priority == PROCUREMENT_PRIORITIES[0][0]
+            for picking in unchanged_pickings
+        )
```

Comment:
Not blocking at all, but if you want `mapped` also supports functions if you want to avoid to write a list comprehension:
```suggestion
        changed_ok = all(
            changed_pickings.mapped(lambda p: p.priority == expected_priority)
        )
        unchanged_ok = all(
            unchanged_pickings.mapped(lambda p: p.priority == PROCUREMENT_PRIORITIES[0][0])
        )
```